.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+- '8'
```

Comment:
you can use a build matrix and test it on both node 8 and node 10

Reply:
done